artemis/modules/joomla_scanner.py

```diff
@@ -34,15 +35,33 @@ def _is_version_old(self, version: str, age_threshold_days: int = 180) -> bool:
             assert cache_result
             data = json.loads(cache_result)
 
+        version_parsed = semver.VersionInfo.parse(version)
+        if version_parsed.major < 3:
+            return True
+
+        is_current_version_old = True  # If we don't find the version on releases list, that means, it's old
         for release in data:
             if release["tag_name"] == version:
-                return (
+                is_current_version_old = (
                     datetime.datetime.utcnow().replace(tzinfo=pytz.utc)
                     - datetime.datetime.fromisoformat(release["published_at"])
                 ).days > age_threshold_days
```

Comment:
mmm, worst code formatting I've seen this week.
```suggestion
                current_time = datetime.datetime.utcnow().replace(tzinfo=pytz.utc)
                published_time = datetime.datetime.fromisoformat(release["published_at"]
                time_difference = (current_time - published_time).days
                is_current_version_old = time_difference > age_threshold_days
```
also, it's common to use
```py
from datetime import datetime
```